common/environment/environment.cpp

```diff
@@ -1648,13 +1646,13 @@ IConstDropZoneInfo * CLocalEnvironment::getDropZoneByAddressPath(const char * ne
 
         if (strncmp(pdropzoneDir, targetFilePath, strlen(pdropzoneDir)) == 0)
```

Comment:
trivial: no need for pdropzonedir var. or strlen, could do strncmp(fullDropZoneDir, targetFilePath, fullDropZoneDir.length())